### DISCUSSION TITLE
Vercel/OG with default font

### DISCUSSION THREAD

**Discussion by idiglove** [vercel/community#5250] [Help]
### Summary

I'm creating a dynamic image with vercel/og. Even with default fonts im getting an error in vercel deployment

```
TypeError: Fetch API cannot load: blob:noto-sans-v27-latin-regular.5dda3fca77107598.ttf
```

I'm in a hobby plan, anyone can pls help? This was working even with custom fonts 3 weeks ago

### Steps to Reproduce

Make a dynamic image in a next js project, use vercel/og, and just add some text.
Deploy in vercel, and run the app

**Comment by amyegan** [comment]
Hey @idiglove. Thanks for reporting this. There was a problem with OG image generation earlier, but it should be working normally now. You can find the incident report at https://www.vercel-status.com/incidents/chv6qxv3szlm

Please let me know if you still see this error. 🙏

**Reply by idiglove** [reply]
Thanks @amyegan ! Yes I subscribed to the incident reports, and found it out. It was resolved pretty fast, thanks for all your work <3

**Comment by jdgamble555** [comment]
This is still a problem. Something to do with the font `noto-sans-v27-latin-regular.ttf` file not loading.

J

For More Reference - https://github.com/vercel/next.js/issues/59853

**Comment by jdgamble555** [comment]
@amyegan - Any fix for this problem!?

https://github.com/vercel/satori/issues/582 

J

**Comment by RickRyan26** [comment]
I've also been waiting on a fix for this @jdgamble555 
Granted I need it for Sveltekit not Next...
Could a fork without the font solve the problem?
